modules/react/icon/lib/AccentIcon.tsx

```diff
@@ -52,6 +60,7 @@ export default class AccentIcon extends React.Component<AccentIconProps> {
         styles={accentIconStyles({color, transparent})}
         size={size}
         iconRef={iconRef}
+        shouldMirror={shouldMirror}
```

Comment:
This could be passed along with `elemProps`, but because we were being explicit with other props, I decided to explicitly destructure and pass it along. I'm open to either option though.